notification/index.azureFunction.js

```diff
@@ -0,0 +1,56 @@
+import notificationHandler from './src/handler/notification/notification.handler.js'
+import { getLogger } from './src/utils/logger.js'
+import utils from './src/utils/commons.js'
+import { getErrorCause, isRecoverableError } from './src/utils/error-utils.js'
+import { getCtpProjectConfig, getAdyenConfig } from './src/utils/parser.js'
+
+const logger = getLogger()
+
+function handleErrorResponse(context, status, message) {
+  context.res = {
+    status,
+    error: {
+      message,
+    },
+  }
+}
+
+export const azureNotificationTrigger = async function (context, req) {
+  const { notificationItems } = req?.body || {}
+  if (!notificationItems) {
+    handleErrorResponse(context, 400, 'No notification received.')
+    return
+  }
+
+  try {
+    for (const notification of notificationItems) {
+      const ctpProjectConfig = getCtpProjectConfig(notification, req.url)
+      const adyenConfig = getAdyenConfig(notification)
+
+      await notificationHandler.processNotification(
+        notification,
+        adyenConfig.enableHmacSignature,
+        ctpProjectConfig
+      )
+    }
+  } catch (err) {
+    const cause = getErrorCause(err)
+    logger.error(
+      {
+        notification: utils.getNotificationForTracking(notificationItems),
+        cause,
+      },
+      'Unexpected exception occurred.'
+    )
+
+    if (isRecoverableError(err)) {
+      handleErrorResponse(context, 500, cause.message)
+      return
+    }
+  }
+
+  context.res = {
```

Comment:
Can you create `handleXXXResponse` so it's consistent like the other azure class in extension?

Reply:
I don't see any benefit here. In extension, we have success response called more than once, So I have extracted common method.

Reply:
> I don't see any benefit here. In extension, we have success response called more than once, So I have extracted common method.

OK, but you know that in extension you also have success response only once :)